stm/vernac_classifier.ml

```diff
@@ -99,15 +99,18 @@ let classify_vernac e =
       VtStartProof(Doesn'tGuaranteeOpacity, idents_of_name i), VtLater
 
     | VernacDefinition (_,({v=i},_),ProveBody _) ->
-       let guarantee = if poly then Doesn'tGuaranteeOpacity else GuaranteesOpacity in
-        VtStartProof(guarantee, idents_of_name i), VtLater
+      let atts = DefAttributes.parse atts in
```

Comment:
`DefAttributes.parse` can cause an unsupported attribute error, is that OK at classify time?

Reply:
Actually I don't agree with using defattributes here, we should only read the attributes we need.

Reply:
Done, thanks!